restcomm/restcomm.http/src/main/java/org/restcomm/connect/http/SupervisorEndpoint.java

```diff
@@ -184,7 +187,7 @@ protected Response registerForUpdates(final String accountSid, final Multivalued
     //Register a remote location where Restcomm will send monitoring updates for a specific Call
     protected Response registerForCallUpdates(final String accountSid, final String callSid, final MultivaluedMap<String, String> data, MediaType responseType) {
         secure(daos.getAccountsDao().getAccount(accountSid), "RestComm:Read:Calls");
```

Comment:
@maria-farooq

Hmm, this line:
> secure(daos.getAccountsDao().getAccount(accountSid), 

 checks the following:

1. User has authenticated
2. Authenticated account is parent of accountSid
3. Authenticated account has the "Restcomm:Read:Calls" permission (as determined by its role)

Then, we also check if this user is a SuperAdmin (i.e. top-level account). 

This means that if a SuperAdmin does not have the "Restcomm:Read:Calls" he won't be granted access. Also, if there is another top-level  account that is not parent (or grant-parent) of the "accountSid" he won't be granted access either. 

Is this ok ? 

Or should we accept that a SuperAdmin has absolute powers on all instance assets and thus rephrase the code as:

// secure()
checkAuthenticatedAccount();
allowOnlySuperAdmin();

I'm not suggesting, i'm just wondering...